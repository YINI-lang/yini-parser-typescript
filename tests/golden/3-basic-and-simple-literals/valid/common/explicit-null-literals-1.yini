^ ExplicitNulls
val1 = null
val2 = NULL
val3 = Null
val4 = nULL

