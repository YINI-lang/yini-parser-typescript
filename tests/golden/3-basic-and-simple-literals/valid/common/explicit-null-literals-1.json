{
    "ExplicitNulls": {
        "val1": null,
        "val2": null,
        "val3": null,
        "val4": null
    }
}
